### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Markdown Formatting Toolbar

### DISCUSSION THREAD

**Discussion by GreenFlux** [dailydotdev/daily#1923] [Feature Requests]
### Concise Description

Add a Markdown toolbar for applying formatting

### Description

Add a Markdown toolbar with support for:
- Bold, Italics
- Ordered/Unordered lists
- Image upload
- 

### Solution Proposed

Unsure what library would work best with the current Daily.dev codebase. But here's a newer option for React that looks interesting. 
https://github.com/ianstormtaylor/slate

### Additional Context

```bash
Improving the writing experience in Daily.dev will encourage more text posts and longer content, instead of users only sharing links to external posts.
```

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @GreenFlux,

Thanks a lot for the suggestion and including a library already.
We will consider it
